app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProjectsController < ApplicationController
+def show 
+@projects = Project.find(params[:id])
+@contestants = @projects.contestants
+end
+
+def update
+  @projects = Projects.find(params[:id])
```

Comment:
This was close; you wanted `Project.find` instead of `Projects` - remember that models are always singular.